apps/docs/docs/troubleshooting/troubleshooting.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 10
+slug: /troubleshooting
+---
+
+# Troubleshooting
+
+When using GOAT, you might face issues for some specific tasks you want to perform. These issues are generally due to computation constraints in GOAT. Computational limits are essential for maintaining server stability and efficiency, ensuring reliable performance. Learning to understand and fix these problems will help you move forward with your analysis. This page is here to guide you through common errors, making it easier to solve them and keep progressing with GOAT. 
```

Comment:
I think it would be sufficient to say "This page is here to guide you through issues or constraints while using GOAT, making it easier to solve them and continue progressing with your analysis."

And then we should also add a tip saying:
"Feel free to contact us for support or any further questions you may have."